src/components/action-popover/action-popover.spec.tsx

```diff
@@ -764,23 +780,19 @@ describe("ActionPopover", () => {
   });
 
   it("validates the children prop", () => {
-    jest.spyOn(global.console, "error").mockImplementation(() => {});
-    const tempWrapper = enzymeMount(
-      <ThemeProvider theme={mintTheme}>
-        <ActionPopover>
-          <ActionPopoverItem onClick={() => {}}>Item</ActionPopoverItem>
-          <p>invalid children</p>
-        </ActionPopover>
-      </ThemeProvider>
-    );
-    tempWrapper.find(MenuButton).simulate("click");
-    jest.runAllTimers();
-    // eslint-disable-next-line no-console
-    expect(console.error).toHaveBeenCalledWith(
-      "Warning: Failed prop type: `ActionPopover` only accepts children of" +
-        " type `WithTheme(ActionPopoverItem)` and `ActionPopoverDivider`.\n    in ActionPopover"
+    expect(() => {
```

Comment:
Can we keep `jest.spyOn(global.console, "error").mockImplementation(() => {});` and `global.console.error.mockReset();` so we're not outputting the invariant errors in the console when the spec files run?

Reply:
Unfortunately not because because with the use of `invariant` we're throwing an error instead of just console logging it and `expect(() => some code that throws an error).toThrow()` is the only way to test it.

Reply:
https://jestjs.io/docs/expect#tothrowerror

`Note: You must wrap the code in a function, otherwise the error will not be caught and the assertion will fail.`